Sources/Instructions/Extra/Default Views/CoachMarkBodyDefaultView.swift

```diff
@@ -24,6 +24,8 @@ public class CoachMarkBodyDefaultView: UIControl,
     public lazy var nextLabel: UILabel = makeNextLabel()
     public lazy var hintLabel: UITextView = makeHintTextView()
     public lazy var separator: UIView = makeSeparator()
+    private let nextLabelPosition: CoachMarkNextLabelPosition?
```

Comment:
I'd set the default position (`.trailing`) directly here.

```suggestion
    private let nextLabelPosition: CoachMarkNextLabelPosition = .trailing
```